src/app/public/modules/country-field/country-field.component.scss

```diff
@@ -29,41 +40,74 @@
 .sky-country-field-input {
   cursor: text;
   display: flex;
-  background-color: $sky-color-white;
-  padding-left: 12px;
+  position: relative;
 
   .sky-country-field-flag {
+    align-items: center;
     margin-bottom: auto;
     margin-top: auto;
-    margin-right: $sky-margin;
+    position: absolute;
+    left: 13px;
+    height: 35px;
+    bottom: 0;
   }
 
   textarea {
-    resize: none;
-    border: 0;
-    overflow: hidden;
-    height: #{$sky-line-height-base}em;
-    box-sizing: content-box;
-    padding: 0;
-    display: inline-flex;
-    background-color: transparent;
-
-    &:focus {
-      outline: none;
-      box-shadow: none;
-    }
+    @include sky-country-field-textarea;
+  }
+}
+
+.sky-country-field-input-with-flag {
+  .sky-country-field-flag {
+    display: flex;
+  }
 
+  .sky-form-control {
+    padding-left: 40px;
   }
 }
 
 .sky-country-field-country-selected, .sky-country-field-country-default {
   @include sky-border(light, bottom);
 }
 
-:host(.ng-invalid.ng-touched) .sky-country-field-input {
+:host(.ng-invalid.ng-touched) .sky-country-field-input textarea {
   @include sky-field-status(invalid);
 }
 
+::ng-deep {
```

Comment:
I thought we were disabling view encapsulation?

Reply:
I considered it, but decided it was less risky to leave it because of the nested components like country field and the potential for unintended side-effects.